frontend/src/lib/components/HTMLElementsDisplay/HTMLElementsDisplay.tsx

```diff
@@ -93,8 +115,16 @@ export function HTMLElementsDisplay({
     const [key] = useState(() => `HtmlElementsDisplay.${uniqueNode++}`)
 
     const logic = htmlElementsDisplayLogic({ checkUniqueness, onChange, key, startingSelector, providedElements })
-    const { parsedSelectors, chosenSelector, chosenSelectorMatchCount, messageStatus, elements } = useValues(logic)
-    const { setParsedSelectors } = useActions(logic)
+    const {
+        parsedSelectors,
+        chosenSelector,
+        chosenSelectorMatchCount,
+        messageStatus,
+        elements,
+        elementsToShowDepth,
+        parsedElements,
+    } = useValues(logic)
+    const { setParsedSelectors, showAdditionalElements } = useActions(logic)
 
     return (
         <div className="flex flex-col gap-1">
```

Comment:
Below this line, we're still using `!!elements.length`, but I believe we should be using `!!parsedElements.length` and remove `elements` from the values used in this component.

Reply:
Addressed in c7085c2